appveyor.yml

```diff
@@ -8,19 +8,19 @@ clone_folder: c:\gopath\src\github.com\urfave\cli
 
 environment:
   GOPATH: C:\gopath
-  GOVERSION: 1.8.x
-  PYTHON: C:\Python36-x64
-  PYTHON_VERSION: 3.6.x
-  PYTHON_ARCH: 64
+  GOVERSION: 1.11.x
 
 install:
-- set PATH=%GOPATH%\bin;C:\go\bin;%PATH%
-- go version
-- go env
-- go get github.com/urfave/gfmrun/...
-- go get -v -t ./...
+  - set PATH=%GOPATH%\bin;C:\go\bin;%PATH%
+  - go version
+  - go env
+  - go get github.com/shurcooL/vfsgen
+  - go get github.com/shurcooL/httpfs/union
+  - go get github.com/urfave/gfmrun/...
+  - go get -v -t ./...
 
 build_script:
-- python runtests vet
-- python runtests test
-- python runtests gfmrun
+  - go run build/build.go generate
+  - go run build/build.go vet
+  - go run build/build.go test
+  - go run build/build.go gfmrun
```

Comment:
go run builds and immediately runs it. The reason I am not using a built CLI is because right now there is no way to go get that package because it does not yet exist on master. This works well as I tested it on my machine

Reply:
I guess my question is why don't we run `go build ./...` as part of CI?

Reply:
I did not try this. Let me try and see how it goes

Reply:
We can try building a binary but it will cause us issues when writing scripts as windows will need .exe and mac/linux won't. There will be a conflict in go generate directive.

Reply:
I think we are misunderstanding each other. I think you are talking about using the library itself for the build script. I am not talking about that. I am saying we generate files, run the tests as part of CI, yet don't actually try to build the code of the library. P

Reply:
That's what I am doing now. I am confused as to what you want me to do. Go build will output a binary however go run will run it without generating a build.

Reply:
I don't see go build anywhere in this file

Reply:
how will go build make the solution any better than what we have currently?

Reply:
Well currently we're not even building the code? So why would building the code make it worse?

Reply:
we are building it. go run first builds the code and then immediately runs it without outputting the binary.

Reply:
I don't see `go build` in either of the CI files, hence why I am confused. How does this get built if there is no go build anywhere? Some CI magic?

Sorry, I know it feels like we are going in circles but just want to understand it.

Reply:
The `go run` builds it! The `go run` command is roughly equivalent of the following:

`go build X.go -o /tmp/random-tmp-folder/exe && /tmp/random-tmp-folder/exe`

so it will build and immediately run.

another way to look at this would be if you could point me to the line of code where we were using something analogous to go build in the old python scripts.

Reply:
But we don't go rum the cli itself, as it's a library. I understand go run builds the flag generator, but I am talking about building the library after the flags are generated to see if it still builds.

Reply:
oh! I agree this is not done but then it wasn't even done before. If we examine the code closely, we will see that if the flags are not generated right, the test cases would fail.

Reply:
It's possible that running tests will not build a certain subpackage.
For example, if I have
`cli/` and  `cli/argparse`, and `cli/argparse` has no `_test.go` files, that package would not be built part of the tests (no tests found), yet might be incompatible with what the flags generated and break when building the library (oppose to when running the library)

Reply:
yes but in our scenario, there is only one different package, altsrc which does have its tests.

Reply:
That doesn't mean it will always be the case. 

Regardless, I agree that it's not a change for this PR, but doing a simple go build would definately up my confidence.